yb-voyager/src/tgtdb/event.go

```diff
@@ -380,10 +380,26 @@ func getMapValuesForQuery(m map[string]*string) []interface{} {
 	return values
 }
 
-func (event *Event) IsUniqueKeyChanged(uniqueKeyCols []string) bool {
-	return event.Op == "u" &&
-		len(uniqueKeyCols) > 0 &&
-		lo.Some(lo.Keys(event.Fields), uniqueKeyCols)
```

Comment:
we may have tested this before, but does our entire conflict detection work properly with case sensitive column names? 
Can we pls add a unit test (either in this or the subsequent PR) for this case.